packages/mantine/src/components/header/Header.tsx

```diff
@@ -46,7 +46,7 @@ export interface HeaderProps extends StylesApiProps<HeaderFactory>, Omit<GroupPr
      *
      * @default 'page'
      */
-    variant?: 'page' | 'modal';
+    variant?: 'page' | 'modal' | 'content';
```

Comment:
I think we should go with primary/secondary instead of duplicating modal. Content doesn't make much sense for me

```suggestion
    variant?: 'primary' | 'secondary';
```